app/boards/shields/contra/boards/nice_nano.overlay

```diff
@@ -0,0 +1,28 @@
+&spi1 {
+	compatible = "nordic,nrf-spim";
+	status = "okay";
+	mosi-pin = <45>; // Data cable is on D15 of Nice!NanoV2
```

Comment:
I see the contra breaks out both D15 and D18. Is D15 over D18 arbitrary, or is there a common pin people pick?

Reply:
I think its just preference, any of the pins can be used for the data cable of the led strip